src/helper/json_helper.cpp

```diff
@@ -160,4 +160,61 @@ dds::parameter json_to_parameter(std::string_view json)
     return json_to_parameter(doc);
 }
 
+std::optional<rapidjson::Value::ConstMemberIterator>
+json_helper::get_field_of_type(
+    const rapidjson::Value &parent_field, const char *key, rapidjson::Type type)
```

Comment:
Use `std::string_view` instead of `const char *`, this applies to all three functions.

Reply:
Done